public/app/plugins/datasource/prometheus/result_transformer.test.ts

```diff
@@ -131,7 +138,7 @@ describe('Prometheus Result Transformer', () => {
       expect(series.data[0].fields[1].name).toEqual('label1');
       expect(series.data[0].fields[2].name).toEqual('label2');
       expect(series.data[0].fields[3].name).toEqual('Value');
-      expect(series.data[0].meta?.preferredVisualisationType).toEqual('table');
+      expect(series.data[0].meta?.preferredVisualisationType).toEqual('rawPrometheus' as PreferredVisualisationType);
```

Comment:
let's avoid unneeded type assertions (same thing for other changes in this file)

```suggestion
      expect(series.data[0].meta?.preferredVisualisationType).toEqual('rawPrometheus');
```